packages/datastream.go

```diff
@@ -177,16 +177,16 @@ func NewDataStream(basePath string, p *Package) (*DataStream, error) {
 		return nil, fmt.Errorf("invalid release: %q", d.Release)
 	}
 
-	pipelineDir := filepath.Join(d.BasePath, "elasticsearch", DirIngestPipeline)
-	paths, err := fs.Glob(filepath.Join(pipelineDir, "*"))
+	pipelineDir := path.Join(d.BasePath, "elasticsearch", DirIngestPipeline)
+	paths, err := fs.Glob(path.Join(pipelineDir, "*"))
```

Comment:
Nit. Maybe we should start using `fsys` in general as abbreviation for filesystems to distingish from `fs` package.